controllers/module_reconciler.go

```diff
@@ -298,6 +316,48 @@ func (r *ModuleReconciler) checkImageExists(ctx context.Context, mod *kmmv1beta1
 	return imageAvailable, nil
 }
 
+func (r *ModuleReconciler) handleDriverContainerServiceAccount(ctx context.Context, mod *kmmv1beta1.Module) error {
+	logger := log.FromContext(ctx)
+
+	sa := &v1.ServiceAccount{}
+	fmt.Println("Before rbacAPI.GetDriverContainerServiceAccount")
+	err := r.rbacAPI.GetDriverContainerServiceAccount(ctx, *mod, sa)
+	if err != nil && !apierrors.IsNotFound(err) {
+		return fmt.Errorf("failed to get the driver container service account: %w", err)
+	}
+	fmt.Println("After rbacAPI.GetDriverContainerServiceAccount")
+
+	opRes, err := controllerutil.CreateOrPatch(ctx, r.Client, sa, func() error {
+		return r.rbacAPI.SetDriverContainerServiceAccountAsDesired(ctx, sa, *mod)
+	})
+
+	if err == nil {
+		logger.Info("Reconciled driver container ServiceAccount", "name", sa.Name, "result", opRes)
+	}
+
+	return err
+}
+
+func (r *ModuleReconciler) handleDriverContainerRoleBinding(ctx context.Context, mod *kmmv1beta1.Module) error {
```

Comment:
Should we group the `RoleBinding` and `ServiceAccount` management into just one function?

Reply:
I was leaning towards this direction myself actually, I'll give it another go. It's been smelling since I pressed the first keystroke :)